gremlin-scala/src/main/scala/gremlin/scala/GremlinScala.scala

```diff
@@ -396,6 +396,8 @@ case class GremlinScala[End, Labels <: HList](traversal: GraphTraversal[_, End])
 
   def times(maxLoops: Int) = GremlinScala[End, Labels](traversal.times(maxLoops))
 
+  def tree() = GremlinScala[Tree[_], Labels](traversal.tree())
```

Comment:
doesn't make a huge difference, but would be good to pass on the type of tree:

`def tree[A]() = GremlinScala[Tree[A], Labels](traversal.tree())`